Package.swift

```diff
@@ -6,12 +6,12 @@ let package = Package(
         .library(name: "MongoSwift", targets: ["MongoSwift"])
     ],
     dependencies: [
-        .package(url: "https://github.com/mongodb/swift-bson", from: "1.0.0"),
-        .package(url: "https://github.com/mongodb/swift-mongoc", from: "1.0.0"),
+        .package(url: "https://github.com/mongodb/swift-bson", .branch("SWIFT-232/update-exported-name")),
```

Comment:
these will change to proper tags once the PRs for those changes over there are accepted, merged, and tagged